tests/util/transactions.ts

```diff
@@ -98,6 +98,7 @@ export async function createContract(
   return {
     rawTx,
     contract,
+    contractAddress,
```

Comment:
I think you can get the contract address from the contract object:

```javascript
contract.options.address
```

Reply:
@joelamouche This came from your integration test. I would like to get that test passing. I think it should be passing. Maybe you can take a look together with @4meta5 ?

Reply:
I put that test in jira 495 because it wasn't passing and I was trying to reduce the scope to get this PR to a merge-able state. If you need the test in this PR, feel free to add it back and go for it.